### PR TITLE
Add support of Graylog GELF Alerter

### PR DESCRIPTION
**Pull request by malinkinsa**
## Checklist

- [x] I have reviewed the [contributing guidelines](https://github.com/jertel/elastalert2/blob/master/CONTRIBUTING.md).
- [x] I have included unit tests for my changes or additions.
- [x] I have successfully run `make test-docker` with my changes.
- [x] I have manually tested all relevant modes of the change in this PR.
- [x] I have updated the [documentation](https://elastalert2.readthedocs.io).
- [x] I have updated the [changelog](https://github.com/jertel/elastalert2/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by jertel** [comment]
Thanks for the submission. I recommend looking at the `tests/alerters/` directory, as you'll probably find a template there to use for adding unit test coverage.

**Comment by malinkinsa** [comment]
Yep, till the end of the week will do it

**Comment by malinkinsa** [comment]
@jertel
Done

**Comment by jertel** [comment]
Thanks for adding the unit tests. Will you please also update the [schema.yaml](https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml) file, with the expect parameter data types? Search for "discord" or "alerta" for examples of what is needed.

**Comment by nsano-rururu** [comment]
http_ignore_ssl_errors → gelf_http_ignore_ssl_errors
timeout → gelf_timeout

**Comment by malinkinsa** [comment]
> update the [schema.yaml](https://github.com/jertel/elastalert2/blob/master/elastalert/schema.yaml) file

Ready, i hope it's correct.

@nsano-rururu many thanks for your edits, with them it is better.

**Review by nsano-rururu — DISMISSED** [review]

File: elastalert/alerters/gelf.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        self.fields = self.rule.get('gelf_payload', {})
+        self.headers = {
+            'Content-Type': 'application/json'
+        }
+        self.gelf_version = self.rule.get('gelf_version', '1.1')
+        self.gelf_log_level = self.rule.get('gelf_log_level', 5)
+        self.additional_headers = self.rule.get('gelf_http_headers')
+        self.ca_cert = self.rule.get('gelf_ca_cert')
+        self.http_ignore_ssl_errors = self.rule.get('http_ignore_ssl_errors', False)
+        self.timeout = self.rule.get('timeout', 30)
```
[the remark this answers is not in the document]
Comment (malinkinsa) [reply elastalert/alerters/gelf.py]: About `gelf_version` - i didn't want to add a specified version list to schemas. The latest version is 1.1 and it is default now and this version was released in 2013. 

About `gelf_log_level` - good point. I'm adding a range now.

**Review by nsano-rururu — APPROVED** [review]

**Pull request merged**